model-server/src/main/kotlin/org/modelix/model/server/handlers/ui/RepositoryOverview.kt

```diff
@@ -31,7 +36,21 @@ class RepositoryOverview(private val repoManager: IRepositoriesManager) {
         application.routing {
             get("/repos") {
                 call.respondHtmlTemplate(PageWithMenuBar("repos/", "..")) {
-                    headContent { title("Repositories") }
+                    headContent {
+                        title("Repositories")
+                        script(type = "text/javascript") {
+                            unsafe {
+                                +"""
+                                    function removeBranch(repository, branch) {
+                                        if (confirm('Are you sure you want to delete the branch' + branch +' of repository' +repository + '?')) {
+                                            fetch('../v2/repositories/' + repository + '/branches/' + branch, { method: 'DELETE'})
+                                            .then( _ => location.reload())
```

Comment:
I hope we never have an error response from the DELETE endpoint. That would probably be invisible to the user now without opening the browser console.